modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerView.vue

```diff
@@ -23,45 +23,49 @@
       </tr>
       <tr
         class="apos-table__row"
-        v-for="row in rows"
-        :key="row._id"
+        v-for="item in items"
+        :key="item._id"
         :class="{'is-selected': false }"
       >
         <td class="apos-table__cell">
           <AposCheckbox
-            v-if="checkboxes[row._id]"
-            :field="checkboxes[row._id].field"
-            :value="checkboxes[row._id].value.data"
-            :status="checkboxes[row._id].status"
-            :choice="checkboxes[row._id].choice"
-            :id="row._id"
+            v-if="item._id"
+            :field="{
```

Comment:
This replaced the previous checkbox generator function. Note that `status` and `value` are no longer passed in since they weren't used.